cpp/oneapi/dal/algo/decision_forest/common.hpp

```diff
@@ -594,6 +598,17 @@ class descriptor : public detail::descriptor_base<Task> {
         return *this;
     }
 
+    /// Engine method for the random numbers generator used by the algorithm
+    /// @remark default = engine_method::mt2203
+    engine_type get_engine_method() const {
```

Comment:
1) Is mt2203 a default method?
2) I have a concern about the naming. Maybe it would be better to rename to `get_engine_type` as the returned object is `engine_type`?

Reply:
fixed, thanks!